samples/counter/ios-shared/src/commonMain/kotlin/com/example/redwood/counter/ios/IosButton.kt

```diff
@@ -24,7 +24,7 @@ import platform.UIKit.UIControlStateNormal
 import platform.UIKit.UIView
 import platform.objc.sel_registerName
 
-internal class IosButton : Button<UIView> {
+class IosButton : Button<UIView> {
```

Comment:
Until we get #536 for iOS:

```suggestion
// NOTE: This class must be public for the click selector to work.
class IosButton : Button<UIView> {
```